app/controllers/paginable/templates_controller.rb

```diff
@@ -49,7 +49,7 @@ def customisable
       when 'not-customised'
         templates = Template.latest_customizable.where.not(family_id: customizations.collect(&:customization_of))
     end
-    paginable_renderise partial: 'customisable', scope: templates.includes(:org), locals: { action: 'customisable', customizations: customizations }
+    paginable_renderise partial: 'customisable', scope: templates.joins(:org), locals: { action: 'customisable', customizations: customizations }
```

Comment:
why not 
```
templates.joins(:org).includes(:org)
```
will permit the search to work but also will pre-fetch the orgs for the view since it uses the org.name for every template...